azext_iot/central/params.py

```diff
@@ -122,6 +156,15 @@ def load_central_arguments(self, _):
             "[Stringified JSON Example: {'a': 'b'}] ",
         )
 
+    with self.argument_context("iot central device create") as context:
+        context.argument(
+            "device_id",
+            options_list=["--device-id", "-d"],
+            help="Provide a unique identifier for the device."
+            " A case-sensitive string (up to 128 characters long) of ASCII 7-bit alphanumeric characters plus"
+            " certain special characters: - . + % _ # * ? ! ( ) , : = @ $ '",
```

Comment:
nit - might be good to specify the regex we allow

Reply:
Leave this here as this is the same description as official documents that Ankit asked.